rosbag2_transport/src/rosbag2_transport/recorder.cpp

```diff
@@ -162,12 +164,12 @@ void Recorder::subscribe_topics(
 {
   for (const auto & topic_with_type : topics_and_types) {
     subscribe_topic(
-      {
-        topic_with_type.first,
-        topic_with_type.second,
-        serialization_format_,
-        serialized_offered_qos_profiles_for_topic(topic_with_type.first)
-      });
+        {
```

Comment:
I don't know why, but `uncrustify` started complaining about this in my local env